util/dvsim/testplanner.py

```diff
@@ -8,33 +8,38 @@
 import argparse
 import sys
 
-from testplanner import testplan_utils
+import Testplan
 
 
 def main():
     parser = argparse.ArgumentParser(
         description=__doc__,
         formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument(
-        'testplan',
-        metavar='<hjson-file>',
-        help='input testplan file (*.hjson)')
-    parser.add_argument(
-        '-r',
-        '--regr_results',
-        metavar='<hjson-file>',
-        help='input regression results file (*.hjson)')
-    parser.add_argument(
-        '--outfile',
-        '-o',
-        type=argparse.FileType('w'),
-        default=sys.stdout,
-        help='output HTML file (without CSS)')
+    parser.add_argument('testplan',
+                        metavar='<hjson-file>',
+                        help='input testplan file (*.hjson)')
+    parser.add_argument('-s',
+                        '--sim_results',
+                        metavar='<hjson-file>',
+                        help='input simulation results file (*.hjson)')
+    parser.add_argument('--outfile',
+                        '-o',
+                        type=argparse.FileType('w'),
+                        default=sys.stdout,
+                        help='output HTML file (without CSS)')
     args = parser.parse_args()
     outfile = args.outfile
 
     with outfile:
```

Comment:
This is confusing me: I thought outfile was a string, so the "with" should wrap the file creation, which is not here.

Reply:
The arg type is already a file stream opened in write mode.

Reply:
I see... I have never use "with" on an already opened file. Apparently in such case it will close on exit from with. Thanks.

Reply:
Please resolve